sorting_visualizer/sort.py

```diff
@@ -78,8 +78,33 @@ def mergesort(A):
 
         counter = 2 * counter
 
-def quicksort(A):
-    return
+# quicksort is a recursive algorithm
+def quicksort(A, low, high):

```

Comment:
This won't work. You need to create generators using `yield`. Also, note that you can only use iterative solutions. Recursive solutions don't exactly return generator objects at each iteration.